home/templatetags/home_tags.py

```diff
@@ -1,16 +1,17 @@
 from django import template
-from home.models import Footer
+import home.models as model
 
 register = template.Library()
 
 @register.inclusion_tag('home/tags/footer.html', takes_context=True)
 def footer(context):
     return {
-        'footer': Footer.objects.first(),
+        'footer': model.Footer.objects.first(),
```

Comment:
I think it's better to leave this as-is. `Footer.objects.first()`

Reply:
@ehmadzubair could you add the reason why you would leave this as it was? There doesn't seem to be any significant difference between the two.

Reply:
done

Reply:
In my experience, a couple of reasons:
- I think importing models as a whole module can create confusions in future when importing from multiple modules. For example, `import users.models as users_models`. This might result in confusing imports in multiple files. In another file, there could be `import users.models as models`.
- Importing the exact model class (`from home.models import model1, model2, model3`) is the de-facto import style in most of the tutorials, documentation and open source code.

Does this make sense?